Sources/SUIComponents/Atoms/SeparatorView.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public extension Components.Atoms {
+    struct SeparatorView: View {
+        let colour: Color
+        public init(colour: Color = .Semantic.divider) {
```

Comment:
I don't think we should use the English spelling TBH

Reply:
Converted to simplified English as requested